tests/javaobject/cli/PositionKey.java

```diff
@@ -26,7 +26,7 @@
   private long positionId;
 
   static {
-     Instantiator.register(new Instantiator(javaobject.cli.PositionKey.class, (byte) 77) {
+     Instantiator.register(new Instantiator(javaobject.cli.PositionKey.class, (byte) 21) {
```

Comment:
I don't know why it casts this literal to a `byte`, the method takes an `int`. Remove the cast.

Reply:
Done.